conjugate_map/conjCalcFunctions.py

```diff
@@ -1,251 +1,306 @@
 # functions for computing geomagnetic conjugate points.
 
 # Importing packages
-import datetime as dt
-import os
-
 import aacgmv2
-from geopack import geopack as gp 
+import datetime as dt
+from geopack import geopack as gp
 from geopack import t89
 import gpxpy
 import gpxpy.gpx
 import numpy as np
+import os
 import pandas as pd
-import plotly.express as px
-import plotly.graph_objects as go
 
-############################################################################################################################### 
 
-def findconj(lat, lon, ut = dt.datetime.utcnow(), is_verbose = False, method = 'auto', limit = 60):
-    
+###############################################################################
+def findconj(lat, lon, ut=dt.datetime.utcnow(),
```

Comment:
```suggestion
def findconj(lat, lon, ut=dt.datetime.now(tz=dt.timezone.utc),
```

`utcnow` has been deprecated 😿